salt/states/proxy.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+'''
+Allows you to manage proxy settings on minions
+=======================
+
+Setup proxy settings on minions
+
+.. code-block:: yaml
+
+    192.168.1.4:
+      proxy.managed:
+        - port: 3128
+        - bypass_domains:
+            - localhost
+            - 127.0.0.1
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+# Import salt libs
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'proxy'
+
+
+def __virtual__():
+    '''
+    Only work on Mac OS
```

Comment:
Seems counterintuitive to say "Only work on mac OS" and afterwards accept `__grains__['os'] == 'Windows']`

I'm confused...